governance/polls/DAO Masons Poll Exercise 2 - README.md

```diff
@@ -0,0 +1,115 @@
+# DAO Masons - Poll Exercise 2
```

Comment:
Generally good, feel free to look through other candidates responses for some other takes and possible complications here.